quickstart/migrate-etcd.md

```diff
@@ -0,0 +1,139 @@
+---
+Title: ETCD Migration Procedure
+---
+
+By following the given steps, you can successfully migrate ETCD from one node to another during maintenance activities like node drain etc., ensuring the continuity and integrity of the ETCD data.
+
+:::note
+Take a snapshot of the ETCD. Click [here](https://etcd.io/docs/v3.3/op-guide/recovery/) for the detailed documentation.
```

Comment:
Should use 3.5 link: https://etcd.io/docs/v3.5/op-guide/recovery/

Reply:
we will not be using snapshot here, this is just to be on the safer side.